src/frontend/scss/objects/_course_glimpses.scss

```diff
@@ -75,6 +75,9 @@ $course-glimpse-content-padding-sides: 0.7rem !default;
   @include media-breakpoint-up(lg) {
     @include sv-flex(1, 0, calc(25% - #{$r-course-glimpse-gutter * 2}));
   }
+  @include media-breakpoint-down(lg) {
+    max-width: 100%;
+  }
```

Comment:
This change impacts all course glimpse all over the CMS. Did you check the impact of this change on Search page, home page, organization detail page, etc. You can easily compare the difference between our local version and the online demo version (https://demo.richie.education)

Reply:
There what's happen on the course page, look's ok to me:
![image](https://github.com/openfun/richie/assets/139848/2b8420eb-aafc-40a5-bbe3-418d5de516c2)

Reply:
i didn't find a blog post with glimse on our demo site but all other pages are ok.